src/persistance/metadataWriter.ts

```diff
@@ -0,0 +1,20 @@
+import { Metadata } from "../metadata/MetadataGenerable";
+import { META_DATA_LOCATION } from "../constants";
+
+export default class MetadataWriter {
+  private _fs: any;
```

Comment:
Avoid using `any` type as much as possible in TS. The main point of using TS is to enforce type checks at compile time. `any` makes TS code as good as JS.

Reply:
What should be the type for the "fs" module?